test/code/plugins/in_sudo_tail_systest.rb

```diff
@@ -0,0 +1,63 @@
+require_relative '../../../source/ext/fluentd/test/helper'
+require 'fluent/test'
+require_relative '../../../source/code/plugins/in_sudo_tail'
+
```

Comment:
Test files that end with `_systest` communicate with a server. It does not seem to be the case here so change it to `_test`.